### PR TITLE
Backend basics

### PR DESCRIPTION
**Pull request by phuchle**
I added in a new ```npm start``` command and set up the basic routing for the express endpoints.  I think this is a good base to start developing more backend stuff!   When you run ```npm start ``` you should see two servers boot up, one at localhost:3005 (server) and one at localhost:3000 (client).  

Right now there's only the root and /api on localhost:3005.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3005;
+const indexRoutes = require('./routes');
+const apiRoutes = require('./routes/api');
+
+// Eventually we will need to refer to the build folder for production
+// It would look like: 
+// if (process.env.NODE_ENV === "production") {
+//   app.use(express.static("client/build"));
```
Comment (jwu910) [inline server/app.js:10]: When we get there, are we setting up another dev script for say like a generic `npm run build` and `npm run build:dev`?

**Review by rainrivas — DISMISSED** [review]

**Review by jwu910 — DISMISSED** [review]
:trollface:

**Review by jwu910 — APPROVED** [review]
:100:

**Review by rainrivas — APPROVED** [review]

File: server/app.js (hunk 1 above)
Comment (phuchle) [reply server/app.js:10]: React-scripts has a build command for the React side of things so we can use that.  I don't think we'll need a build step for the back end, but if we do we can cross that bridge then.

Comment (jwu910) [reply server/app.js:10]: Yeah I think we covered this last night. Thanks!

**Pull request merged**